src/config.sh

```diff
@@ -35,6 +35,11 @@ function config_main()
     exit 0
   fi
 
+  if [[ "$#" == 0 ]]; then
+    show_configurations
+    exit 0
```

Comment:
`show_configurations` may return a failure code. Use `$?` instead of 0, and replace `exit` with `return`.